src/jit/codegenlinear.h

```diff
@@ -42,6 +42,10 @@
     void                genPutArgStk(GenTreePtr treeNode);
     unsigned            getBaseVarForPutArgStk(GenTreePtr treeNode);
 
+#ifdef FEATURE_UNIX_AMD64_STRUCT_PASSING
```

Comment:
I left it in the target specific code. Some targets would actually not need such method. So the specific target will decide whether it needs it and define it properly. If defined in codegencommon.coo every target (even the ones that don't need it) will have to have it. The calling convention are vastly different, so I have a feeling code reuse between targets will not be possible. And if it is, we can always move it as appropriate.